proto/common/common.proto

```diff
@@ -19,6 +19,12 @@ enum ValueKind {
   BUFFER = 3;
 }
 
+message Data {
```

Comment:
`Data` seems too generic. It might be nice for this to be `LabeledData` or something like that.